lib/robots/dor_repo/gis_assembly/generate_mods.rb

```diff
@@ -96,28 +93,28 @@ def geometry_type_ogrinfo(shp_filename)
         QSEC = 'ʺ'
         QMIN = 'ʹ'
         QDEG = "\u00B0"
-        def dd2ddmmss_abs(f)
-          dd = f.to_f.abs
-          d = dd.floor
-          mm = ((dd - d) * 60)
-          m = mm.floor
-          s = ((mm - mm.floor) * 60).round
-          if s >= 60
-            m += 1
-            s = 0
+        def dd2ddmmss_abs(orig_val)
+          orig_val_abs_float = orig_val.to_f.abs
+          degrees = orig_val_abs_float.floor
+          minutes_float = ((orig_val_abs_float - degrees) * 60)
+          minutes = minutes_float.floor
+          seconds = ((minutes_float - minutes) * 60).round
+          if seconds >= 60
+            minutes += 1
+            seconds = 0
           end
-          if m >= 60
-            d += 1
-            m = 0
+          if minutes >= 60
+            degrees += 1
+            minutes = 0
```

Comment:
same questions as for same method in other class.  La la la la I can't HEAR you.